resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -14,7 +14,7 @@ if [ -r "$restartfile" ];  then
 
         ((timeout--))
         if [ "$timeout" = 0 ]; then
-            echo "Reload timed out after <%= EZbake::Config[:start_timeout] %> seconds"
+            echo "$timeout" | awk '{printf "Reload timed out after %.3f secs\n", $1/10}'
```

Comment:
You'd need to be doing math either way. I mentioned this in another comment, but it may have gotten lost.

"I would recommend doing this in ruby over shell. <%= EZbake::Config[:start_timeout] / 10 %>"